resolve/cpu/CMakeLists.txt

```
@@ -17,12 +17,7 @@ set(ReSolve_CPU_HEADER_INSTALL
 
 # First create dummy backend
 add_library(resolve_backend_cpu SHARED ${ReSolve_CPU_SRC})
-target_link_libraries(resolve_backend_cpu PRIVATE resolve_logger)
-
-target_include_directories(resolve_backend_cpu INTERFACE
-    $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}>
-    $<INSTALL_INTERFACE:include>
-)
+target_link_libraries(resolve_backend_cpu PUBLIC resolve_logger)
```

Could we make logger dependency private?
```suggestion
target_link_libraries(resolve_backend_cpu PRIVATE resolve_logger)
```